recipe/meta.yaml

```
@@ -23,6 +23,7 @@ requirements:
   run:
     - python >=3.6
     - uritools
+    - google-cloud-storage >=1.31.0
```

This should only be a constraint, not a hard dependency:
```suggestion
  run_constrained:
    - google-cloud-storage >=1.31.0
```